### PR TITLE
Fix link & typos

### PR DESCRIPTION
**Pull request by ifdattic**

### CODE REVIEW

**Comment by ciaranmcnulty** [comment]
Thanks!

**Comment by ifdattic** [comment]
Glad to help!

File: docs/manual/installation.rst [hunk 1]
```diff
@@ -31,10 +31,10 @@ Then install phpspec with the composer install command:
 
     $ composer install
 
-Follow instructions on `the composer website <https://getcomposer.org/download/`_
+Follow instructions on `the composer website <https://getcomposer.org/download/>`_
 if you don't have it installed yet.
 
-Phpspec with its dependencies will be installed inside the ``vendor`` folder
+phpspec with its dependencies will be installed inside the ``vendor`` folder
```
Comment (GrahamCampbell) [inline docs/manual/installation.rst]: ?

Comment (GrahamCampbell) [inline docs/manual/installation.rst]: A sentence is still required by English to start with an uppercase letter.

Comment (ifdattic) [inline docs/manual/installation.rst]: Yes, but `phpspec` is a name & most of the time it's used in lowercase letters (see the beginning of the file for example)

Comment (ciaranmcnulty) [inline docs/manual/installation.rst]: There's no consistent style guide for how to style 'PhpSpec'. Some brands insist on lowercase even at the start of a line, English experts think that's silly.... 

*Sigh*, I'll write a style guide

**Pull request merged**